checkmates/data_checks/datacheck_meta/data_check_message_code.py

```diff
@@ -58,6 +58,9 @@ class DataCheckMessageCode(Enum):
     TARGET_LOGNORMAL_DISTRIBUTION = "target_lognormal_distribution"
     """Message code for target data with a lognormal distribution."""
 
+    SKEWED_DISTRIBUTION = "skewed_distribution"
```

Comment:
I'd assume we shouldn't just because all of these distributions would be normalized using the same method, should probably change this code to a more general term though in that case